lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -328,9 +344,9 @@ Function Choco-Install
                 -skipscripts $skipscripts -source $source -installargs $installargs `
                 -packageparams $packageparams -allowemptychecksums $allowemptychecksums `
                 -ignorechecksums $ignorechecksums -ignoredependencies $ignoredependencies `
-                -allowdowngrade $allowdowngrade -proxy_url $proxy_url `
-                -proxy_username $proxy_username -proxy_password $proxy_password `
-                -allowprerelease $allowprerelease
+                -allowdowngrade $allowdowngrade -allowprerelease $allowprerelease `
```

Comment:
Please keep this section the same so we preserve the git history. Doesn't seem like there is a reason to change this.